l10n_es_facturae/views/report_facturae.xml

```diff
@@ -289,35 +287,41 @@
                         <t t-if="version not in ('3_2', '3_2_1')">
                             <InvoiceDescription
                                 t-length="2500"
-                                t-if="invoice.comment"
-                                t-esc="invoice.comment"
+                                t-if="move.narration"
+                                t-esc="move.narration"
                             />
                             <ReceiverTransactionReference t-length="20" t-if="False" />
                             <FileReference t-length="20" t-if="False" />
                             <ReceiverContractReference t-length="20" t-if="False" />
                         </t>
                     </InvoiceIssueData>
                     <TaxesOutputs>
-                        <t t-foreach="invoice.tax_line_ids" t-as="tax_line">
-                            <Tax t-if="tax_line.tax_id.amount &gt;=0">
-                                <TaxTypeCode t-esc="tax_line.tax_id.facturae_code" />
+                        <t t-foreach="move.line_ids" t-as="tax_line">
+                            <Tax t-if="tax_line.tax_line_id">
```

Comment:
Esto darà problemas con los IRPFs, que son withheld, deberias comprobar:
`tax_line.tax_line_id and tax_line.tax_line_id.amount &gt;=0`